pkg/client/result.go

```diff
@@ -147,6 +144,12 @@ func ObjectsFromResult(res *Result) map[string]*Objects {
 			}
 		}
 
+		if row[1] == nil || row[2] == nil {
+			continue
+		}
+		name := row[1].(string)
+		object_type := row[2].(string)
```

Comment:
If you are available please help change to `objectType` since go uses `camelCase` not `snake_case`

Reply:
I know ;) ... I tried to stick as close to the coding style I found.